src/components/video-capture/video-capture.html

```diff
@@ -0,0 +1,24 @@
+<transition name="modal">
+<div class="modal-mask" @click="close">
+    <div style="height:30%"></div>
```

Comment:
would this be better as fullscreen? especially on mobile?

Reply:
Yep, it needs tidy up. Something in the CSS means the view is not sized correctly.